op-node/rollup/driver/state.go

```diff
@@ -262,10 +266,8 @@ func (s *state) eventLoop() {
 
 		case <-l2BlockCreationReqCh:
 			s.snapshot("L2 Block Creation Request")
-			// only create blocks if we processed all L1 contents
-			progress := s.derivation.Progress() // TODO: add conf depth
-			if progress.Origin != s.l1Head || !progress.Closed {
-				s.log.Warn("not creating block, node is not synced", "progress_origin", progress.Origin, "progress_closed", progress.Closed, "head_l1", s.l1Head)
+			if !s.idleDerivation {
```

Comment:
I think the logic is right, but the name is kinda feels like a double negative with this check & is a little confusing.